src/Common/StackTrace.cpp

```diff
@@ -30,7 +44,7 @@ std::string signalToErrorMessage(int sig, const siginfo_t & info, [[maybe_unused
             /// Print info about address and reason.
             if (nullptr == info.si_addr)
                 error << "Address: NULL pointer.";
-            else
+            else if (show_addresses.load(std::memory_order_relaxed))
```

Comment:
Maybe we can print address anyway if its within the first 4KB? Sometimes it's useful to know offset when debugging segfaults with null pointer